src/client/datascience/notebook/intellisense/jupyterCompletionProvider.ts

```diff
@@ -157,9 +192,43 @@ export class JupyterCompletionProvider implements CompletionItemProvider {
             metadata: {}
         };
     }
+
+    private async getPylanceCompletions(
+        document: TextDocument,
+        position: Position,
+        context: CompletionContext,
+        cancelToken: CancellationToken
+    ) {
+        if (document.notebook) {
```

Comment:
Should probably return [] if language is not python.

Reply:
Document selector already handles this.